src/codelabs/dart-cheatsheet.md

```diff
@@ -559,58 +554,72 @@ void main() {
     final product = obj.product;
     
     if (product != 30) {
-      errs.add('The product property returned $product instead of the expected value (30).'); 
+      errs.add('The product property returned $product \n instead of the expected value (30).'); 
     } 
-  } on UnimplementedError {
-    _result(false, ['Tried to use MyClass.product but failed. Did you implement the method?']);
-    return;
   } catch (e) {
-    _result(false, ['Tried to use MyClass.product, but encountered an exception: ${e.runtimeType}.']);
+    print('Tried to use MyClass.product, but encountered an exception: \n ${e.runtimeType}.');
     return;
   }
 
   try {
     obj.incrementValue1();
     
     if (obj.value1 != 3) {
-      errs.add('After calling incrementValue, value1 was ${obj.value1} instead of the expected value (3).'); 
+      errs.add('After calling incrementValue, value1 was ${obj.value1} \n instead of the expected value (3).'); 
     } 
-  } on UnimplementedError {
-    _result(false, ['Tried to use MyClass.incrementValue1 but failed. Did you implement the method?']);
-    return;
   } catch (e) {
-    _result(false, ['Tried to use MyClass.incrementValue1, but encountered an exception: ${e.runtimeType}.']);
+    print('Tried to use MyClass.incrementValue1, but encountered an exception: \n ${e.runtimeType}.');
     return;
   }
 
   try {
     final joined = obj.joinWithCommas(['one', 'two', 'three']);
     
     if (joined != 'one,two,three') {
-      errs.add('Tried calling joinWithCommas([\'one\', \'two\', \'three\']) and received $joined instead of the expected value (\'one,two,three\').'); 
+      errs.add('Tried calling joinWithCommas([\'one\', \'two\', \'three\']) \n and received $joined instead of the expected value (\'one,two,three\').'); 
     } 
-  } on UnimplementedError {
-    _result(false, ['Tried to use MyClass.joinWithCommas but failed. Did you implement the method?']);
-    return;
   } catch (e) {
-    _result(false, ['Tried to use MyClass.joinWithCommas, but encountered an exception: ${e.runtimeType}.']);
+    print('Tried to use MyClass.joinWithCommas, but encountered an exception: \n ${e.runtimeType}.');
     return;
   }
 
   if (errs.isEmpty) {
-    _result(true);
+    print('Success!');
   } else {
-    _result(false, errs);
+    errs.forEach(print);
   }
 }
-{$ end test.dart $}
-{$ begin hint.txt $}
-For the product, you can just multiply the three values together.
-For incrementValue1, you can use the increment operator (++).
-For joinWithCommas, try using the join method found in the List class.
-{$ end hint.txt $}
+{$ end main.dart $}
 ```
 
+<details markdown="1">
+ <summary>
+   <b>Solution for arrow syntax example</b>
+  </summary>
+
+  For the product, you can just multiply the three values together.
+  For `incrementValue1`, you can use the increment operator (`++`).
+  For `joinWithCommas`, try using the `join` method found in the `List` class.
```

Comment:
Since this is in the solution rather than a hint now, "try" doesn't make sense anymore.

```suggestion
  For `joinWithCommas`, use the `join` method found in the `List` class.
```